apps/cowswap-frontend/src/modules/bridge/components/BridgeAccordionSummary.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Component to display bridge-related information in the collapsed state of an accordion.
+ * Shows the bridge time estimation and protocol icons next to the fee amount.
+ */
+
+import { ReactNode } from 'react'
+
+import { displayTime } from '@cowprotocol/common-utils'
+
+import { ProtocolIcons } from 'common/pure/ProtocolIcons'
+
+import { BridgeProtocolConfig } from '../types'
+interface BridgeAccordionSummaryProps {
+  bridgeEstimatedTime?: number
+  bridgeProtocol?: BridgeProtocolConfig
+  children: ReactNode // This would be the FiatAmount component
+}
+
+/**
+ * Component to display bridge-related information in the collapsed state of an accordion
+ * Shows the bridge time estimation and protocol icons next to the fee amount
+ */
+export function BridgeAccordionSummary({ bridgeEstimatedTime, bridgeProtocol, children }: BridgeAccordionSummaryProps) {
```

Comment:
Need to split `modules/bridge/components` into `modules/bridge/containers` and `modules/bridge/pure`

Reply:
Addressed in https://github.com/cowprotocol/cowswap/pull/5639